packages/pagination/README.md

```diff
@@ -99,6 +99,21 @@ return (
 );
 ```
 
+## createInfiniteScroll
+```tsx
+export default function App() {
+  // fetcher: (page: number) => Promise<T[]>
+  const [pages, loaderDirective] = createInfiniteScroll(fetcher);
+  return (
+    <div>
+      <For each={pages()}>
+        {item => <h4>{item}</h4>}
+      </For>
+      <h1 use:loaderDirective>Loading...</h1>
```

Comment:
You should probably use `<Show when={!ended}>` for the loader. Also, use: already shows that it is a directive, which is why I suggested `asLoader` as an ideal name for the directive.